ci/build.js

```diff
@@ -19,6 +19,7 @@ const ci = new CI();
 
 ci.context();
 
+// Import PGP key into 'gpg' config + configure non-interactive mode
 ci.sh('echo $GPG_PRIVATE_KEY | base64 --decode | gpg --batch --import');
 ci.sh('echo "pinentry-mode loopback" > /home/circleci/.gnupg/gpg.conf');
```

Comment:
Is this run in a docker image? If not, then I think this has the potential to pollute the environment.